examples/modules/plus-number/CMakeLists.txt

```diff
@@ -0,0 +1,109 @@
+cmake_minimum_required(VERSION 3.2 FATAL_ERROR)
+
+project("addon")
+
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY_RELEASE ${PROJECT_SOURCE_DIR}/bin)
+set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY_DEBUG ${PROJECT_SOURCE_DIR}/bin)
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY_RELEASE ${PROJECT_SOURCE_DIR}/bin)
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY_DEBUG ${PROJECT_SOURCE_DIR}/bin)
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY_RELEASE ${PROJECT_SOURCE_DIR}/bin)
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY_DEBUG ${PROJECT_SOURCE_DIR}/bin)
+
+# Require Cxx14 features
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+# Set symbol visibility to hidden by default.
+# Napa shared library shares a few classes with napa-binding.node with different compile definition,
+# exposing the same symbols from both shared libraries may cause improper behaviors under gcc.
+set(CMAKE_CXX_VISIBILITY_PRESET hidden)
+set(CMAKE_VISIBILITY_INLINES_HIDDEN)
+
+# Prefix an suffix of Library name for different OS.
+if (WIN32)
```

Comment:
How about using cmake find_library, it should deal with this automaktically.

---
In reply to: [133273556](https://github.com/Microsoft/napajs/pull/38#discussion_r133273556) [](ancestors = 133273556)